docs/managing_translations_i18n.md

```diff
@@ -0,0 +1,22 @@
+# Managing translations (i18n)
+
+## The workflow
+
+Decidim uses [Crowdin](https://crowdin.com/) to manage the translations.
+
+- Whenever someone [adds a new translation key](https://github.com/decidim/decidim/pull/1814/files#diff-c78c80097da59920d55b3f462ca21afaR177) to _Decidim_, _Crowdin_ gets notified and the new content is available to be translated from [Crowdin's Decidim dashboard](https://crowdin.com/project/decidim).
+- When a translator translates any key from Crowdin, it automatically creates a [PR in Github](https://github.com/decidim/decidim/pull/2207), adding the keys in the corresponding _yaml_ files.
+- 🌈
+
+## Adding a new language
+
+- Setup the new language in [_Crowdin's Decidim project_](https://crowdin.com/project/decidim) (or open an issue on Github asking an admin to do that).
+- Translate at least one key from every engine, so, your _yaml_ files are not empty. The easiest way to do this is to automatically translate and sync all the content. Later you'll be able to fix the content that wasn't properly translated.
+- Add [Foundation Datepicker](https://github.com/najlepsiwebdesigner/foundation-datepicker/tree/master/js/locales)'s translations ([PR](https://github.com/decidim/decidim/pull/2039)).
+- Add Select2 translations ([PR](https://github.com/decidim/decidim/pull/2214)).
+- Add the new language to `available_locales` ([PR](https://github.com/decidim/decidim/pull/1991)).
+- Announce the new language in the Readme ([PR](https://github.com/decidim/decidim/pull/2125)).
+
+## Test the new language
+
+To test the new language in a local development app, first, you need to add the new language to the [initializer template](https://github.com/decidim/decidim/blob/master/lib/generators/decidim/templates/initializer.rb#L9). After that, [create the `development_app` and start the server](https://github.com/decidim/decidim#browse-decidim).
```

Comment:
Instead of that, you could:

1. Generate the development app annd `cd` into it
1. Change the `config/initializer/decidim.rb` file and add your locale to `Decidim.available_locales`
1. `rake db:drop db:setup` to drop, create, load schema and seed the DB

It's almost the same, but you don't regenerate the whole app and you don't need to modify `decidim` files (so `git status` keeps clean)

Reply:
Thank you @mrcasals , this valuable info should also be available publicly.

Somehow the 

$ bin/rails decidim:check_locales

mentioned at

https://github.com/decidim/decidim/blob/master/docs/getting_started.md

didn't work for me at all, and I wasn't able to add Ukrainian to the stable release.

Reply:
Hi @Digharatta! Can you open an issue with this problem so we can track it? 😄

Reply:
Yes, please!

That task would be also interesting to mention here. `bin/rails decidim:check_locales` inside the development app should help confirming that you are done with translating everything!

Reply:
Yes, but translation comes from Crowdin so you'd need to wait for Crowdin send a PR, checkout to that branch and check the locales... Isn't it better to just check the Crowdin dashboard? 🤔

Reply:
You're right, the task is good for applications upgrading `decidim`, but not for this use case :+1:

Reply:
> Instead of that, you could:

> Generate the development app annd cd into it
> Change the config/initializer/decidim.rb file and add your locale to Decidim.available_locales
> rake db:drop db:setup to drop, create, load schema and seed the DB

OK! I'll suggest that approach. 👍